MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.h

```diff
@@ -201,6 +201,9 @@ class MVKRenderPassAttachment : public MVKBaseObject {
     /** Returns whether this attachment should be cleared in the subpass. */
     bool shouldUseClearAttachment(MVKRenderSubpass* subpass);
 
+    /** Returns stencil attachment load op */
```

Comment:
Doc comments should be complete sentences. Also, this one doesn't have any information that couldn't be inferred from the method name.

Reply:
Added clarifying comment that is a complete sentence :)